@navikt/core/react/src/form/form-summary/FormSummaryHeader.tsx

```diff
@@ -1,28 +1,31 @@
 import React, { forwardRef } from "react";
 import { useRenameCSS } from "../../theme/Theme";
+import { CompositionWarning } from "../../util/composition-warning";
 
+/**
+ * Header slot for section heading and actions in `FormSummary`.
+ */
 export interface FormSummaryHeaderProps
   extends React.HTMLAttributes<HTMLDivElement> {
-  /**
-   * Must include `<FormSummary.Heading>` and optionally `<FormSummary.EditLink>`.
-   */
   children: React.ReactNode;
 }
 
 export const FormSummaryHeader = forwardRef<
-  HTMLDivElement,
+  HTMLHeadingElement,
```

Comment:
I believe this was correct before

```suggestion
  HTMLDivElement,
```